### ISSUE TITLE
orchestra\testbench required by ApplicationResolver

### ISSUE THREAD

**Issue by AJenbo** [nunomaduro/larastan#845]
`ApplicationResolver` requires `orchestra\testbench` but it has only been marked as a suggested package.

https://github.com/nunomaduro/larastan/blob/f74befd87b7278285cbf6f1597509cbfaca46bf8/src/ApplicationResolver.php#L18

https://github.com/nunomaduro/larastan/blob/c103da52d6c455b05d3a50cda7fc9a648a5b8d5a/composer.json#L33

I'm not sure why this is the case as there is no description of why it is a suggestion, but for a package (not an application) PHPStan fails to run with Larastan loaded

```
> phpstan analyze
Note: Using configuration file phpstan.neon.
PHP Fatal error:  Trait 'Orchestra\Testbench\Concerns\CreatesApplication' not found in vendor/nunomaduro/larastan/src/ApplicationResolver.php on line 16
Fatal error: Trait 'Orchestra\Testbench\Concerns\CreatesApplication' not found in vendor/nunomaduro/larastan/src/ApplicationResolver.php on line 16
```

**Comment by szepeviktor** [comment]
It is very simple. From the README

> Using Larastan for analysing Laravel packages? You may need to install orchestra/testbench.

1. You have an Application -> Use Larastan
2. You have a package -> Install testbench -> Use Larastan

It is okay for you?

**Comment by AJenbo** [comment]
Lets put that directly in the description so it's easier to find for people that read the code before the docs :)

https://github.com/nunomaduro/larastan/pull/847

**Comment by szepeviktor** [comment]
Resolved by #846
